content/en/blog/_posts/2023-08-15-kubernetes-1.28-blog.md

```diff
@@ -253,7 +248,7 @@ Kubernetes v1.28 is available for download on [GitHub](https://github.com/kubern
  
 ## Release Team
 
-Kubernetes is only possible with the support, commitment, and hard work of its community. Each release team is comprised of dedicated community volunteers who work together to build the many pieces that make up the Kubernetes releases you rely on. This requires the specialized skills of people from all corners of our community, from the code itself to its documentation and project management.
+Kubernetes is only possible with the support, commitment, and hard work of its community. Each release team is composed of dedicated community volunteers who work together to build the many pieces that make up the Kubernetes releases you rely on. This requires the specialized skills of people from all corners of our community, from the code itself to its documentation and project management.
```

Comment:
comprised reads correctly here. I wouldn't make this change.

Reply:
ok, sorry for my French